packages/lsp/src/connection.ts

```diff
@@ -120,6 +120,7 @@ class ServerRequestHandler<
 export const Provider: { [key: string]: keyof lsp.ServerCapabilities } = {
   TEXT_DOCUMENT_SYNC: 'textDocumentSync',
   COMPLETION: 'completionProvider',
+  //INLINE_COMPLETION: 'inlineCompletionProvider',
```

Comment:
Should we remove this dead case?

```suggestion
```

Reply:
Yes, this will be added in a future PR once the LSP API stabilises.